third_party/blink/renderer/farbling/brave_session_cache.cc

```diff
@@ -76,6 +83,11 @@ blink::WebContentSettingsClient* GetContentSettingsClientFor(
   blink::WebContentSettingsClient* settings = nullptr;
   if (!context)
     return settings;
+  // Avoid blocking fingerprinting in WebUI pages.
+  const String protocol = context->GetSecurityOrigin()->Protocol();
+  if (protocol != "https" && protocol != "http") {
```

Comment:
`url::kHttpsScheme` `url::kHttpScheme`

Reply:
this doesn't seem like the right place for this check though. Seems like it should be part of `GetBraveFarblingLevel`, etc... and same with the existing checks below. Can we move them in a follow-up?

Reply:
Fixed, and will follow up here: https://github.com/brave/brave-browser/issues/24424